### PR TITLE
Add config option to show or hide the invite link for groups

### PR DESCRIPTION
**Pull request by Neriderc**
Added option SHOW_GROUP_INVITATION_LINKS to show or hide invite links for groups.

When set to false, invite link will not show on group settings pages, and reset API will not work, however, existing link will still work.

Also added CLI command to list invite links for groups, and a command to reset the invite link.

This is covers part of #50 

This PR relies on #73

### CODE REVIEW

File: api/src/Controller/Group/ResetInviteKey.php [hunk 1]
```diff
@@ -43,8 +43,9 @@ public function index(
         string $id,
         #[CurrentUser] User $currentUser
     ): Response {
-        $this->denyAccessUnlessGranted('ROLE_USER');
```
Comment (nrobinaubertin) [inline api/src/Controller/Group/ResetInviteKey.php:46]: We still want to deny access to this endpoint if the user is not logged in. I think that line 46 should be kept

Comment (Neriderc) [reply api/src/Controller/Group/ResetInviteKey.php:46]: Ah I didn't intend to remove this, good catch. Have restored it.

File: api/src/Service/Mailer.php [hunk 2]
```diff
@@ -6,6 +6,7 @@
 use Psr\Log\LoggerInterface;
 use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\Mime\Email;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
```
Comment (nrobinaubertin) [inline api/src/Service/Mailer.php:9]: This now needs to be removed too

Comment (Neriderc) [reply api/src/Service/Mailer.php]: Ah right, fixed

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**